.ruby-version

```diff
@@ -1 +1 @@
-2.2.2
\ No newline at end of file
+2.1.5
```

Comment:
It's totally fine to add a new (older) ruby version and test against it, but don't change the .ruby-version file, this should only be changed upwards..